llvm/test/CodeGen/X86/apx/asm-constraint.ll

```diff
@@ -1,21 +1,27 @@
 ; Check r16-r31 can not be used with 'q','r','l' constraint for backward compatibility.
-; RUN: not llc < %s -mtriple=x86_64-unknown-unknown -mattr=+egpr 2>&1 | FileCheck %s
+; RUN: not llc < %s -mtriple=x86_64-unknown-unknown -mattr=+egpr 2>&1 | FileCheck %s --check-prefix=ERR
```

Comment:
Is line 2 same as line 4? If so, remove line 2.

Reply:
line 2 is to check stderr, line 4 is to check stdout, I don't have a good way to check them both in on RUN line. Any idea?

Reply:
Really? IMO, line 4 is checking stderr too.

Reply:
no, it didn't add `2>&1`

Reply:
oh, I got your point. I messed with line 5

Reply:
[e752556](https://github.com/llvm/llvm-project/pull/92338/commits/e752556c06ac25d905c6e642bdcb5e9244db5da3)